tests/test_gw_repeater.sh

```diff
@@ -55,8 +55,8 @@ check_wsl() {
     # Docker fails to route traffic from the host to the containers
     # if the IP is anything but what was allocated by the daemon
     # when the container was created
-    GW_EXTRA_OPT="--expose ${GW_UCC_PORT} --publish 127.0.0.1::${GW_UCC_PORT} --ipaddr 0.0.0.0"
-    RP_EXTRA_OPT="--expose ${RP_UCC_PORT} --publish 127.0.0.1::${RP_UCC_PORT} --ipaddr 0.0.0.0"
+    GW_EXTRA_OPT=(--expose "${GW_UCC_PORT}" --publish 127.0.0.1::"${GW_UCC_PORT}" --ipaddr 0.0.0.0)
```

Comment:
The commit message explains that the shellcheck advice is not taken, but it doesn't explain what you did instead.